test/spec/features/dragging/HoverFixSpec.js

```diff
@@ -11,6 +11,7 @@ import { createCanvasEvent as canvasEvent } from '../../../util/MockEvents';
 
 import dragModule from 'lib/features/dragging';
 
+var LOW_PRIORITY = 500;
```

Comment:
I wonder why this is necessary. What is the behavior we'd like to support?

* default _move_ listeners are never called -> hook into `drag.move` with high priority
* default _move_ listeners are maybe called, depending on registration order -> keep implementation as is

My suggestion would be: 

* change the test case to verify that listeners with normal priority are called once only
* update the implementation (listen for `drag.move` with high priority)

Reply:
Good catch, just updated this!